hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/contract/s3a/S3AContract.java

```diff
@@ -63,6 +68,22 @@ public S3AContract(Configuration conf, boolean addContractResource) {
     }
   }
 
+  /**
+   * Skip S3AFS initialization if S3-CSE and S3Guard are enabled.
+   *
+   * @throws IOException throw IOE if it is not due to S3-CSE and S3Guard
```

Comment:
doesn't any more; will throw `AssumptionViolatedException`

Reply:
didn't understand this, This means we either skip or throw IOE if not caused by CSE, S3Guard incompatibility. 
Do you mean, I should mention either IOE or `AssumptionViolatedException`(skip)?

Reply:
just cut the IOE reference